### ISSUE TITLE
Can't import from node_modules

### ISSUE THREAD

**Issue by dnish** [fourseven/meteor-scss#250]
Hey,
I try to import a .scss file within client/imports.scss:

    @import '{}/node_modules/normalize-scss/sass/_normalize.scss';

App crashes with error:

```
/client/imports.scss: Scss compiler error: File to import: {}/node_modules/normalize-scss/sass/_normalize.scss not found in file: /Users/dennis/Desktop/Entwicklung/myapp/myapp/{}/client/imports.scss
```

But the path exists within the node_modules directory. Any idea what's going wrong?

**Comment by pilarArr** [comment]
I'm having the same problem

**Comment by sebakerckhof** [comment]
Can you try again with v4.5.3?

**Comment by dhulme** [comment]
Working for me in v4.5.3, thank you!

**Comment by dnish** [comment]
When I try to use the normalize-scss plugin, I still get the same error.

**Comment by dnish** [comment]
Oh okay, after installing it, I needed to restart my Meteor app completely. Now it works, thank you very much :)

**Comment by pilarArr** [comment]
Not working for me on v4.5.4 but my issue is related more to issue #219 since I'm trying to import a css file. I tried moving the css file to the stylesheets folder and had the same error, but when I changed the extension to css it worked.

**Comment by sebakerckhof** [comment]
@pilarArr Yep, that seems to be related to the other issue, so I'll close this one.

**Comment by pilarArr** [comment]
Hi @sebakerckhof, the thing is that issue #219 is closed, in there you say that is a duplicate of #195, then issue #195 is also closed, and you point to issue #165 to continue the discussion there, on issue #165 you point to meteor/meteor#6037 which is also closed.

But issue #165 is off topic in the sense that it refers specifically to Meteor 1.3. 
I'm using 1.5 and still have this issue. Should I open a new issue?
